examples/seed.js

```diff
@@ -245,7 +245,7 @@ const seedActivities = next => {
 const seedProcedures = next => {
   procedures = Procedure.fake(activities.length);
   _.forEach(activities, (activity, index) => {
-    procedures[index].activity = activity;
+    procedures[index].activity = activities[0];
```

Comment:
@Mloweedgar  why have you change to this?

Reply:
@BenMaruchu that is not supposed to be there, I forgot to remove it during debbuging, I will fix it asap